src/sugar3/graphics/popwindow.py

```diff
@@ -0,0 +1,244 @@
+# Copyright (C) 2016 Abhijit Patel
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# Some examples:
+#   Viewsource window
+#   Viewhelp window
+#
+# Can be further implemented in activities like
+#   Physics activity for Load Sample window.
+#   Read activity for the bookmark window.
+#   Bibliography for "new item" winodow.
+#   Pippy activity for Load example window.
+
```

Comment:
Can you add a docstring with an example in here?

Maybe just add a simple example:  making a PopWindow containing a GtkLabel and with an extra toolbutton in the toolbar.